projects/plugins/crm/src/automation/commons/triggers/transactions/class-transaction-updated.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Jetpack CRM Automation Transaction_Updated trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Transaction_Updated class.
+ *
+ * @since $$next-version$$
+ */
+class Transaction_Updated extends Base_Trigger {
+
+	/**
+	 * Get the slug name of the trigger.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/transaction_updated';
+	}
+
+	/**
+	 * Get the title of the trigger.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string
+	 */
+	public static function get_title(): string {
+		return __( 'Transaction Updated', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the description of the trigger.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string
+	 */
+	public static function get_description(): string {
+		return __( 'Triggered when a transaction is updated', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the category of the trigger.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string
+	 */
+	public static function get_category(): string {
+		return __( 'transaction', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Listen to this trigger's target event.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return void
+	 */
+	protected function listen_to_event() {
+		add_action(
+			'jpcrm_transaction_update',
```

Comment:
```suggestion
			'jpcrm_transaction_updated',
```

The event verb should be in the past.